<ui version="4.0" >
 <class>LayoutDialog</class>
 <widget class="QDialog" name="LayoutDialog" >
  <property name="windowModality" >
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Page Layout</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Page Size</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="3" column="0" >
       <widget class="QLabel" name="label_7" >
        <property name="text" >
         <string>Orentation</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" >
       <widget class="QRadioButton" name="landscapeRB" >
        <property name="text" >
         <string>Landscape</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QRadioButton" name="portraitRB" >
        <property name="text" >
         <string>Protrait</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QDoubleSpinBox" name="widthSpinBox" >
        <property name="suffix" >
         <string> in</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>Width</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QDoubleSpinBox" name="heightSpinBox" >
        <property name="suffix" >
         <string> in</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>Height</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2" >
       <widget class="QComboBox" name="pagesizecombo" />
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2" >
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>5</hsizetype>
       <vsizetype>4</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>Margins</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="5" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="5" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="4" >
       <widget class="QDoubleSpinBox" name="rightSpinBox" >
        <property name="suffix" >
         <string> in</string>
        </property>
        <property name="singleStep" >
         <double>0.100000000000000</double>
        </property>
        <property name="value" >
         <double>0.900000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="4" >
       <widget class="QDoubleSpinBox" name="leftSpinBox" >
        <property name="suffix" >
         <string> in</string>
        </property>
        <property name="singleStep" >
         <double>0.100000000000000</double>
        </property>
        <property name="value" >
         <double>0.900000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QDoubleSpinBox" name="bottomSpinBox" >
        <property name="suffix" >
         <string> in</string>
        </property>
        <property name="singleStep" >
         <double>0.100000000000000</double>
        </property>
        <property name="value" >
         <double>0.900000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QDoubleSpinBox" name="topSpinBox" >
        <property name="suffix" >
         <string> in</string>
        </property>
        <property name="singleStep" >
         <double>0.100000000000000</double>
        </property>
        <property name="value" >
         <double>0.900000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="3" >
       <widget class="QLabel" name="label_5" >
        <property name="text" >
         <string>Left</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3" >
       <widget class="QLabel" name="label_6" >
        <property name="text" >
         <string>Right</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Top</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_4" >
        <property name="text" >
         <string>Bottom</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="2" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton" >
       <property name="text" >
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pagesizecombo</tabstop>
  <tabstop>heightSpinBox</tabstop>
  <tabstop>widthSpinBox</tabstop>
  <tabstop>portraitRB</tabstop>
  <tabstop>landscapeRB</tabstop>
  <tabstop>topSpinBox</tabstop>
  <tabstop>bottomSpinBox</tabstop>
  <tabstop>leftSpinBox</tabstop>
  <tabstop>rightSpinBox</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>LayoutDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>278</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel" >
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>LayoutDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>369</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel" >
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
